### PR TITLE
consolidating command/blueprint print logic

### PR DESCRIPTION
**Pull request by kellyselden**
Part of my quest to clean up code.
Command and blueprint printing were remarkably similar, so I extracted into a print command utility. There were only minor inconsistencies that resulted in "breaking" changes. By breaking I mean the help text changed a little bit, for instance whether "(Required)" came before or after "(Default: value)".

### CODE REVIEW

File: lib/models/blueprint.js [hunk 1]
```diff
@@ -1231,76 +1232,19 @@ Blueprint.prototype.printBasicHelp = function(verbose) {
   } else {
     output += this.name;
 
-    var options = this.anonymousOptions;
+    output += printCommand.call(this, initialMargin, true);
```
Comment (stefanpenner) [inline lib/models/blueprint.js]: lets install the shared command on the respective prototypes, so they can be invoked off `this` rather then via call.

Comment (kellyselden) [inline lib/models/blueprint.js]: Can do. Is there a performance reason I don't know about, or just style issue?

Comment (stefanpenner) [inline lib/models/blueprint.js]: both (although the performance in this case doesn't matter)

**Comment by kellyselden** [comment]
@stefanpenner comment addressed. Installed it as a private member function. Also changed the test stubbing as a result.

**Comment by stefanpenner** [comment]
:+1:

**Comment by kellyselden** [comment]
@homu r+

**Pull request merged**